odl/util/normalize.py

```diff
@@ -262,6 +261,76 @@ def normalized_index_expression(indices, shape, int_to_slice=False):
     return tuple(indices)
 
 
+def normalized_nodes_on_bdry(nodes_on_bdry, length):
```

Comment:
Ok fine. The more I think of this odl.util things, we really need to get #463 done so users dont need to use `odl.util` in any extent. That would keep them away from it.